api/v1alpha1/directivebreakdown_types.go

```diff
@@ -64,6 +64,15 @@ type AllocationSetConstraints struct {
 	// Labels is a list of labels is used to filter the Storage resources
 	Labels []string `json:"labels,omitempty"`
 
+	// Scale is a hint for the number of allocations to make based on a 1-10 value
+	// +kubebuilder:validation:Minimum:=1
+	// +kubebuilder:validation:Maximum:=10
+	Scale int `json:"scale,omitempty"`
+
+	// Count is the number of the allocations to make
+	// +kubebuilder:validation:Minimum:=1
+	Count int `json:"count,omitempty"`
```

Comment:
Does omitempty set this to 1 since that's the minimum?

Reply:
It won't show up in the yaml, but in the go struct it's 0. When you marshal the yaml for a create, it will omit the value in the yaml it sends to the API Server if "Count" is 0 in the struct.